src/content/docs/apis/nerdgraph/examples/nerdgraph-api-notifications-channels.mdx

```diff
@@ -0,0 +1,671 @@
+---
+title: "NerdGraph tutorial: applied intelligence channels"
+tags:
+  - Notifications
+  - Notifications and Nerdgraph
+  - Channels
+  - Notification Channels
+metaDescription: 'For New Relic applied intelligence: how you can list, create, update, and delete channels with our NerdGraph API.'
+redirects:
+  - /docs/apis/nerdgraph/examples/nerdgraph-api-notifications-destinations
+---
+
+You can manage your applied intelligence channels using our NerdGraph API.
+
```

Comment:
@zuluecho9 I got some feedback from the AI team that we may need to clarify that channels in this doc refer to notifications using destinations and not the legacy notification channels used by alerts.